Catappult/appcoins-wallet-android
# [APPC-4119] VIP refactor v3

**What does this PR do?**

refactor VIP screen and add onboarding screen when users become vip

**Database changed?**

  No

**How should this be manually tested?**

 With a VIP wallet, check if promotion card has vip mark, verify countdown on referral card and when click on the card, you should be redirected to vip screen

in the first time you are a vip, you should be able to see a vip.onboarding screen 

**What are the relevant tickets?**

  Tickets related to this pull-request: [APPC-4119](https://aptoide.atlassian.net/browse/APPC-4119)

**Code Review Checklist**



[APPC-4119]: https://aptoide.atlassian.net/browse/APPC-4119?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
